pyet/radiation.py

```diff
@@ -346,6 +346,57 @@ def makkink(tmean, rs, pressure=None, elevation=None, k=0.65, clip_zero=True):
     return pet.rename("Makkink")
 
 
+def makkink_knmi(tmean, rs, clip_zero=True):
+    """Potential evapotranspiration calculated according to The Royal Netherlands
+    Meteorological Institute (KNMI)
+
+    Parameters
+    ----------
+    tmean : float/pandas.Series/xarray.DataArray
+        average day temperature [°C]
+    K : float/pandas.Series/xarray.DataArray
+        incoming solar radiation [MJ m-2 d-1]
+
+    Returns
+    -------
+    float/pandas.Series/xarray.DataArray containing the calculated
+            Potential evapotranspiration [mm d-1].
+
+    Examples
+    --------
+    >>> pet_mak = makkink_knmi(tmean, rs)
+
+    Notes
+    ----
+
+    This method is only applicable to the Netherlands (~sea level). Calculating
+    the Makkink evaporation with :cite:t:`makkink_testing_1957` formula is more
+    suitable for general purposes.
+
```

Comment:
Could you perhaps state what the difference is? Some empirical value ?

Reply:
Updated the notes.